docs-internal/db.md

```diff
@@ -0,0 +1,364 @@
+# Helidon Database Proposal
+
+Provide an API that enables reactive database support to be used with Helidon SE.
+
+## Proposal
+
+The Helidon DB is an abstraction layer over
+- database configuration
+- statement configuration
+- statement processing
+- handling of results
+
+The Helidon DB in addition supports
```

Comment:
Maybe instead "The Helidon DB also supports".

Reply:
ok